docker/app/Dockerfile

```diff
@@ -18,8 +18,9 @@ WORKDIR /app
 
 COPY Gemfile Gemfile.lock ./
 
-RUN bundle install --without production
+RUN bundle config set without 'production'
```

Comment:
@aaron-collier has requested that this be changed to *only* include production dependencies, which is reasonable IMO. I plan to do that in a follow-up PR since this one is already sizable.

Reply:
why?